presto-docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -83,6 +83,7 @@ Property Name                                               Description
 ``pinot.broker-authentication-type``                        Pinot authentication method for broker requests. Allowed values are ``NONE`` and ``PASSWORD`` - defaults to ``NONE`` which is no authentication.
 ``pinot.broker-authentication-user``                        Broker username for basic authentication method.
 ``pinot.broker-authentication-password``                    Broker password for basic authentication method.
+``pinot.query-options``                                     Pinot query-related options.
```

Comment:
Also, make this a session config, so you can disable it at runtime.

Reply:
Also, add some sample values for the query option.

Reply:
Another question here is how generic you want this feature to be. 
Either you add a boolean connector and session config for `pinot.enable-null-handling`, or we can use query-option, which is more advanced but also easy error prune one :p

Reply:
@xiangfu0 Prefer to keep `pinot.query-options` config as is since it mimics what Pinot supports and I have already tested with multiple options and it works (e.g. `pinot.query-options=skipUpsert:true,enableNullHandling=true`). Please note that these options are case-sensitive. I will add more details to the description.